packages/clickhouse-server.yaml

```diff
@@ -9,24 +16,21 @@ homepage: "https://clickhouse.com"
 license: "Apache"
 section: "database"
 priority: "optional"
+maintainer: "ClickHouse Dev Team <packages+linux@clickhouse.com>"
+deb:
+  fields:
+    Source: clickhouse
 
-conflicts:
-- clickhouse-keeper
+# Package specific content
 replaces:
 - clickhouse-server-common
 - clickhouse-server-base
 provides:
+- clickhouse-keeper
```

Comment:
This is the relevant change.

PS. I didn't understand everything else in the diff, but it looks good to me.